integration/nwo/fsc/node/note_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package node
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type DummySDK struct {
```

Comment:
for documentation purposes, can you add another SDK here
```
type DummySDK2 struct {}
type DummySDK3 struct {}
```

and then test
```
node.AddSDKWithBase(&DummySDK{}, &DummySDK2{})
node.AddSDKWithBase(&DummySDK3{})
```
This way we know what to expect when we read it and we explain all possible ways to use it.

Reply:
Extended the test. For the second part, the code there doesn't use reflection. It is just combining strings.